reference/cd-github.md

```diff
@@ -0,0 +1,92 @@
+---
+title: GitHub Workflow (Preview)
+---
+
+<style>
+    .img-bordered {
+        border: 2px solid rgba(0,0,0,0.15);
+    }
+    .move-right {
+        margin-left: 32px;
+    }
+    .move-right-small {
+        margin-left: 8px;
+    }
+</style>
+
+Pulumi's GitHub workflow integration is a feature which shows the results of Pulumi stack
+updates on GitHub. Showing you any potential infrastructure changes on Pull Requests,
+as well as recording stack changes made with each push.
+
+> GitHub workflow integration is currently in preview, and we will be expanding the feature set
+> over the comming weeks and months. If you have any feedback, suggestions, or bug reports, please
+> let us know in the [Pulumi Community Slack](https://slack.pulumi.io/).
+
+
```

Comment:
Since the installation instructions are below, an `<a href>` to it here might be nice. E.g.

[See below]() for details on how to install the Pulumi GitHub App into your account.

Reply:
Done.